### PR TITLE
Implement All Remaining Features the Original SAM Does

### PR DESCRIPTION
**Pull request by wgpierce**
Implement only showing locked/unlocked achievements.
Implement launching app in separate window.

This is branched off feat-timed-modifications so I could continue working. It'll apply cleanly on dev after feat-timed-modifications is merged to dev.

### CODE REVIEW

File: src/types/Achievement.h [hunk 1]
```diff
@@ -49,6 +49,7 @@ typedef struct Achievement_t Achievement_t;
 struct AchievementChange_t {
     std::string id;
     bool achieved;
+    uint64_t num;
```
Comment (PaulCombal) [inline src/types/Achievement.h]: Can you comment above what this is about?

**Review by PaulCombal — CHANGES_REQUESTED** [review]
Can you give an example usage for CLI timed achievement unlocking under the options list on the README?

Also if you want to show it off, you can write a little paragraph about timed achievements ;)

**Comment by PaulCombal** [comment]
Really awesome work! I can't believe you put so much effort in that, congrats! There's just the tw little things I pointed out in the review, but also about the new "timed" window, can you add a little bit of padding on the edges of the window? 5-10px let's say. 
Also, it looks like there would be conflicts when merging, I think this also needs to be sorted out, maybe by merging dev into the feature branch first?
Other than that, fantastic! Tell me what you think

File: src/types/Achievement.h (hunk 1 above)
Comment (wgpierce) [reply src/types/Achievement.h]: Done.

**Comment by wgpierce** [comment]
> Can you give an example usage for CLI timed achievement unlocking under the options list on the README?

Done.

> Also if you want to show it off, you can write a little paragraph about timed achievements ;)

Done. Added a "features" section too for significant features.

> about the new "timed" window, can you add a little bit of padding on the edges of the window? 5-10px let's say.

Done.

> Also, it looks like there would be conflicts when merging, I think this also needs to be sorted out, maybe by merging dev into the feature branch first?

Resolved.

Any other changes you want to make to the README? Do you want to bump the screenshots? I can if you want (though I use a dark theme).

**Review by PaulCombal — APPROVED** [review]

**Pull request merged**